src/main/java/frc/robot/subsystems/intake/IntakeIOReal.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.MotionMagicExpoTorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import frc.robot.Constants;
+
+public class IntakeIOReal implements IntakeIO {
+
+    private final CANSparkMax spinMotor = new CANSparkMax(0, CANSparkLowLevel.MotorType.kBrushless);
+    private final CANSparkMax centerMotor =
+            new CANSparkMax(0, CANSparkLowLevel.MotorType.kBrushless);
+    private final TalonFX angleMotor = new TalonFX(21);
+    private final MotionMagicExpoTorqueCurrentFOC positionRequest =
+            new MotionMagicExpoTorqueCurrentFOC(0);
+
+    public IntakeIOReal() {
+        IntakeConstants.ANGLE_CONFIGURATION.Feedback.SensorToMechanismRatio = IntakeConstants.ANGLE_GEAR_RATIO;
+
+        IntakeConstants.ANGLE_CONFIGURATION.Feedback.RotorToSensorRatio = 1;
+
+        angleMotor.getConfigurator().apply(IntakeConstants.ANGLE_CONFIGURATION.Slot0);
+
+        spinMotor.restoreFactoryDefaults();
+        centerMotor.restoreFactoryDefaults();
+        spinMotor.setIdleMode(CANSparkMax.IdleMode.kCoast);
+        centerMotor.setIdleMode(CANSparkMax.IdleMode.kCoast);
+        spinMotor.enableVoltageCompensation(Constants.NOMINAL_VOLTAGE);
+        centerMotor.enableVoltageCompensation(Constants.NOMINAL_VOLTAGE);
+        spinMotor.setInverted(true);
+        centerMotor.setInverted(true);
+        for (int i = 1; i <= 6; i++) {
+            spinMotor.setPeriodicFramePeriod(CANSparkLowLevel.PeriodicFrame.fromId(i), 50_000);
+            centerMotor.setPeriodicFramePeriod(CANSparkLowLevel.PeriodicFrame.fromId(i), 50_000);
```

Comment:
50000? That's a little steep.

Reply:
The robot works at 50 hz, and Falcons work at 1000 hz.

Reply:
The intake goes brrrr

Reply:
86dc3ad832e59c01d8597a47f0258055bfb998ef